recipes-devtools/openocd/openocd_riscv.bb

```diff
@@ -8,3 +8,5 @@ SRC_URI = " \
 "
 
 SRCREV_openocd = "f93ede5401c711e55d9852986aa399c0318efb22"
+
+EXTRA_OECONF_append = " --enable-remote-bitbang"
```

Comment:
this option should either be marked with `_riscv64` overrrides or sent to oe-core

Reply:
Yep, good point. I'll send it upstream.